aleks-kh/home work/personal_data_application_form.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>User information</title>
+  </head>
+  <body>
+    <form action="data_base.html"
+          method="POST"
+          size="200">
+      <fieldset>
+        <legend align="center">Personal information</legend>
+          <div>
+            <label for="first_last name">Your name is</label>
+            <input id="first_last name" type="text" placeholder="Enter your name"/>
+          </div>
+          <br />
+        <div>
+          <label for="select_city">You live in</label>
+          <select id="select_city" name="selection of the residence city">
```

Comment:
Thanks,
I got you. That would be good to mention it in future, because I did not know that's prohibited.